tests/server/app/test_capabilities.py

```diff
@@ -11,9 +13,17 @@
 
 class CapabilitiesTest(ServerTest):
 
+    def setUp(self) -> None:
+        super().setUp()
+        wait_for_server_startup = os.environ.get('WAIT_FOR_STARTUP',
+                                                 '0') == '1'
+        if wait_for_server_startup:
+            time.sleep(10)
+
     def add_extension(self, er: ExtensionRegistry) -> None:
-        er.add_extension(loader=extension.import_component('xcube_geodb_openeo.api:api'),
-                         point=EXTENSION_POINT_SERVER_APIS, name='geodb-openeo')
+        er.add_extension(
+            loader=extension.import_component('xcube_geodb_openeo.api:api'),
+            point=EXTENSION_POINT_SERVER_APIS, name='geodb-openeo')
```

Comment:
If you want to test the actual extensions, you don't need to add them manually. Just register them in your `plugin.py`.

Reply:
I do register them in `plugin.py`, but removing the lines from `test_capabilities.py` leads to a 404 error - the api is not loaded in the test.